src/tests/unittests/WireTests.cpp

```diff
@@ -787,7 +788,7 @@ TEST_F(WireBufferMappingTests, UnmapCalledTooEarlyForRead) {
     nxtBufferUnmap(buffer);
 
     // The callback shouldn't get called, even when the request succeeded on the server side
-    FlushServer();
+    FlushServer(false);
```

Comment:
Does this look like the right expectation? (note also that FlushClient on line 782 succeeds)